controller/src/main/java/org/jboss/as/controller/CapabilityRegistry.java

```diff
@@ -905,6 +909,45 @@ private SatisfactoryCapability findSatisfactoryCapability(String capabilityName,
         return null;
     }
 
+    @Override
+    public Set<String> suggestCapabilities(String referencedCapability,
```

Comment:
I would just rename this method to something else, suggest might be fine for CLI.
but for generic use it should be something like getProviderPoints() vs getPossibleProviderPoints() we already have.
or even getRegistrationPoints()